bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/CommonConnection.java

```diff
@@ -54,11 +51,8 @@
    */
   Configuration getConfiguration();
 
-  /**
-   * Getter for the field <code>options</code>.
-   *
-   * @return a {@link com.google.cloud.bigtable.config.BigtableOptions} object.
-   */
+  /** @deprecated Please use {@link #getBigtableHBaseSettings()}. */
```

Comment:
nit. Can we rename getBigtableHBaseSettings -> getBigtableSettings? The hbase part is implied since this is an hbase connection

Reply:
marking as resolved(address this on #2466)

Reply:
Please update the javadoc